### ISSUE TITLE
Discussion: Finding a new home for docs

### ISSUE THREAD

**Issue by craigpalermo** [priceline/design-system#755]
There has been some talk of moving to a Storybook-only docs solution instead of maintaining the docs site in addition to Storybook. Nothing has been decided officially. Let's use this issue to discuss further.

Related:
- https://github.com/priceline/design-system/issues/730#issuecomment-616824336

**Comment by sdalonzo** [comment]
I think I can be the assignee for at least doing a simple proof-of-concept so that we can evaluate something

**Comment by craigpalermo** [comment]
@sdalonzo Agreed, that makes sense since you have a fuller picture of the potential plans for DSM.
